README.md

```diff
@@ -80,20 +80,23 @@ If you're using Credo and Styler, **we recommend disabling these rules in `.cred
 
 | `Credo` credo | notes |
 |---------------|-------|
-| `Credo.Check.Consistency.MultiAliasImportRequireUse` | (always expands `A.{B, C}`) |
+| `Credo.Check.Consistency.MultiAliasImportRequireUse` | always expands `A.{B, C}` |
 | `Credo.Check.Readability.AliasOrder` | |
 | `Credo.Check.Readability.BlockPipe` | |
-| `Credo.Check.Readability.LargeNumbers` | (goes further than formatter - fixes bad underscores, eg: `100_00` -> `10_000`) |
-| `Credo.Check.Readability.ModuleDoc` | (adds `@moduledoc false`) |
+| `Credo.Check.Readability.PipeIntoAnonymousFunctions` | |
```

Comment:
added some missing checks we're fixing, like this one. removed parens from notes.